api/share/tools.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, List, Union
+from http import HTTPStatus
+
+
+def build_correct_http_response(data: Union[Dict, List],
```

Comment:
```suggestion
def build_correct_http_response(data: Union[dict, list],
```
See my note above on Python version.
It could also be useful to indicate what the dict/list contain (e.g. `list[str]`).
Do you also accept tuple? Maybe use `Sequence`?

Reply:
see previous comment